infrastructure/stacks/capacity_management/apig-organisation.tf

```diff
@@ -0,0 +1,135 @@
+#######
+# Organisations endpoint
+#######
+# TODO
+# Uncomment authorizer and switch authorization to COGNITO_USER_POOLS
+resource "aws_api_gateway_resource" "organisations_resource" {
+  parent_id   = aws_api_gateway_rest_api.cm_rest_api.root_resource_id
+  path_part   = "organisations"
+  rest_api_id = aws_api_gateway_rest_api.cm_rest_api.id
+}
+
+#######
+# REST methods attached to the endpoint - POST PUT GET DELETE
+# links method to apig, resource/endpoint and authorizer
+#######
+resource "aws_api_gateway_method" "organisations_POST" {
```

Comment:
If we make the http_method = ANY and decide that the lambda called (via the integration works out if it was a POST,GET etc) can we strip this down to just one method